quantum/dynamic_macro.h

```diff
@@ -278,10 +286,10 @@ bool process_record_dynamic_macro(uint16_t keycode, keyrecord_t *record)
             /* Store the key in the macro buffer and process it normally. */
```

Comment:
No, you don't need to do that. False is the right return value for both the press and release event.